components/SwipeListView.js

```diff
@@ -50,6 +55,12 @@ class SwipeListView extends Component {
 		this.props.onRowOpen && this.props.onRowOpen(secId, rowId, rowMap);
 	}
 
+	onRowClose(secId, rowId, rowMap) {
```

Comment:
I can't see where this is being used, unless I'm missing something? If that's the case can you remove this function please